dvc/utils/fs.py

```diff
@@ -101,8 +101,13 @@ def move(src, dst, mode=None):
     else:
         shutil.move(src, tmp)
 
-    if mode is not None:
-        os.chmod(tmp, mode)
+    try:
```

Comment:
Can we check the permissions before we move it?

Reply:
Thanks for the feedback, it is better to first do the check.